src/qgis_stac/ui/connection_dialog.ui

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui version="4.0">
```

Comment:
@Fanevanjanahary can you also make the UI responsive, at the moment the dialog components have a fixed layout.